.circleci/config.yml

```diff
@@ -165,6 +165,17 @@ jobs:
       - run: apt-get update -y && apt-get install -y libgflags-dev
       - run: TEST_TMPDIR=/dev/shm && make V=1 -j16 unity_test | .circleci/cat_ignore_eagain
 
+  build-linux-gcc-4-8:
+    docker:
+      - image: ubuntu-1604:201903-01
+    resource_class: xlarge
+    steps:
+      - checkout
+      - run: pyenv global 3.5.2
+      - run: sudo apt-get install gcc-4.8 g++-4.8 && sudo update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-4.8 50 && sudo update-alternatives --install /usr/bin/g++ g++ /usr/bin/g++-4.8 50
```

Comment:
Should not need both update-alternatives and setting CC and CXX below. Although it offers some protection against accidental use of plain 'gcc' and 'g++' commands in build, I think it's more misleading than protective.